### ISSUE TITLE
numpy has no attribute 'int'

### ISSUE THREAD

**Issue by isaacovercast** [dereneaton/ipyrad#510]
Clean install inside binder with numpy 1.24.3
```
ipyrad -p params-wat.txt -s 1234567 -c 1

 -------------------------------------------------------------
  ipyrad [v.0.9.90]
  Interactive assembly and analysis of RAD-seq data
 ------------------------------------------------------------- 
  Parallel connection | jupyter-dereneaton-2dipyrad-2dway44tm8: 1 cores
  
  Step 1: Demultiplexing fastq data to Samples

  Encountered an Error.
  Message: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  Parallel connection closed.
```

**Comment by isaacovercast** [comment]
I replaced np.int with np.int64 in the few places it occurred and also np.bool with np.bool_ in write_outputs.py

**Comment by dylanHco** [comment]
Hey @isaacovercast I just got a similar error using ipyrad v.  0.9.90.. any thoughts? Thanks!

`-------------------------------------------------------------
  ipyrad [v.0.9.90]
  Interactive assembly and analysis of RAD-seq data
 ------------------------------------------------------------- 
  Parallel connection | i51: 6 cores
  
  Step 7: Filtering and formatting output files 
  [####################] 100% 0:00:51 | applying filters       
/rhome/dcohen01/bigdata/.conda/ipyrad3/lib/python3.10/site-packages/ipyrad/assemble/write_outputs.py:321: FutureWarning: In the future `np.bool` will be defined as the corresponding NumPy scalar.
  mask = afilts[:, 0:2].sum(axis=1).astype(np.bool)

  Encountered an Error.
  Message: module 'numpy' has no attribute 'bool'.
`np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  Parallel connection closed.`

**Comment by isaacovercast** [comment]
Hello, yes this was fixed the other day and has gone out in v0.9.92, which I just pushed to bioconda and the new package should be up within 24 hours. Try to update to 0.9.92 tomorrow and it will work. Thanks for reporting!
